client/packages/system/src/Report/components/ReportArgumentsModal.tsx

```diff
@@ -4,6 +4,7 @@ import { JsonData, JsonForm } from '@openmsupply-client/programs';
 import { ReportRowFragment } from '../api';
 import { useDialog } from '@common/hooks';
 import { DialogButton } from '@common/components';
+import { useAuthContext } from 'packages/common/src';
```

Comment:
for the jest tests 🙏 
```suggestion
import { useAuthContext } from '@openmsupply-client/common';
```